src/components/Nav.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
I also did one Navbar, can you check mine that we may decide which one we will keep? I think I keep the original App file that was the entry to the program. Can you check what I did please, also I need the app.css file because I use it for my styling.